action_tests/assert.js

```diff
@@ -114,6 +116,10 @@ const EXPECTED_CLI_CALL_FACTORIES = {
       "series-name",
     ],
   ],
+  "all-hold-the-line-old-cli-version": () => [
```

Comment:
add comment explaining how this test works (that the stub hands back env.TRUNK_CLI_VERSION)